src/components/WebexAvatar/WebexAvatar.test.js

```diff
@@ -9,86 +9,97 @@ import WebexAvatar from './WebexAvatar';
 jest.mock('../hooks/usePerson');
 
 describe('Webex Avatar component', () => {
-  let personID, peopleJSONAdapter;
+  let personID, newPeople, peopleAdapter;
 
   beforeEach(() => {
     [personID] = Object.keys(people);
-    peopleJSONAdapter = new PeopleJSONAdapter(people);
-    global.console.error = jest.fn();
+    newPeople = people;
+    peopleAdapter = new PeopleJSONAdapter(people);
```

Comment:
why did you change the variable name?

Reply:
to be consistent with `WebexActivity.test.js` naming. and I can't change that to `peopleJSONAdapter`, since it's defined at the component level. Adapters prop in webexActivty is an object that contains two adapters.